packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationSettings.tsx

```diff
@@ -58,14 +58,18 @@ const OrganizationProfileSection = () => {
 };
 
 const OrganizationDangerSection = () => {
-  const { organization, membership } = useCoreOrganization();
+  const { organization, membership, membershipList } = useCoreOrganization({
```

Comment:
can we do `membershipList: adminList` to be a bit clearer?

Reply:
Updated!